packages/pigeon/lib/cpp_generator.dart

```diff
@@ -590,7 +590,7 @@ const flutter::StandardMessageCodec& ${api.name}::GetCodec() {
                     indent.write('if ($encodableArgName.IsNull()) ');
                     indent.scoped('{', '}', () {
                       indent.writeln(
-                          'reply(WrapError("$argName unexpectedly null."));');
+                          'reply(flutter::EncodableValue(std::move(WrapError("$argName unexpectedly null."))));');
```

Comment:
Optional: this `move` isn't necessary, because it's already an rvalue since it's not stored in a variable. It won't hurt, it just doesn't really do anything.